modules/packages/LinearAlgebra.chpl

```diff
@@ -2271,9 +2276,7 @@ proc eig(A: [] ?t, param left = false, param right = false)
                    eltType=real);
     var (U, s, Vh) = svd(A);
 
-  ``LinearAlgebraError`` will be thrown if the SVD computation does not
-  converge or an illegal argument, such as a matrix containing a ``NAN`` value,
-  is given.
+  :throws LinearAlgebraError: if the SVD computation does not converge or an illegal argument, such as a matrix containing a ``NAN`` value, is given.
```

Comment:
This line is a little long - for proper formatting, you'll want the part of the description on the second line to align with the first word after the `:`, e.g.

```
  :throws LinearAlgebraError: if the SVD computation does not converge or an
                              illegal argument, such as a matrix containing a
                              ``NAN`` value, is given.
```